### DISCUSSION TITLE
Help with mconfig

### DISCUSSION THREAD

**Discussion by javieraubert** [magma/magma#14435] [Q&A] [answered]
Hi everyone, does anybody knows or have an idea of what could cause this error: "failed to build mconfig: mconfig builder &{service:feg}", recently install new orch for v1.8 and now my AGW cant get their mconfig files. Thanks in advance.

**Comment by sebathomas** [comment]
It seems there is some error while building the mconfig related to the feg services. Is there any more error log output in your orc8r logs?

**Reply by javieraubert** [reply]
Hi @sebathomas thanks for your reply, i just recreate all the orch again, and it seems to be working now. May be it was a bug not sure.

**Comment by javieraubert** [comment, marked as answer]
Recreate allthe orch and the issue seems to be fixed.
